serialization.md

```diff
@@ -732,5 +717,21 @@ Tree Root Helper Functions
     rule #asmTxPrefix (0) => Legacy
     rule #asmTxPrefix (1) => AccessList
 
+    syntax TxData ::= LegacyTxData         ( nonce: Int, gasPrice: Int, gasLimit: Int, to: Account, value: Int, data: ByteArray )
+                    | LegacyProtectedTxData( nonce: Int, gasPrice: Int, gasLimit: Int, to: Account, value: Int, data: ByteArray, chainId: Int )
+                    | AccessListTxData     ( nonce: Int, gasPrice: Int, gasLimit: Int, to: Account, value: Int, data: ByteArray, chainId: Int, accessLists: JSONs )
+ // ---------------------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
This is to help with implementing the new transaction type in EIP-1559 and future transaction types. The parameters for symbols like `#sender` and `#hashUnsignedTx` were becoming quite numerous and were unused in certain cases, so this also tries to cut down on that.

Reply:
Oh wow, this is really nice, makes it much more easy to compare to the EIP RLP spec :)